src/components/navigation/UserMenu.tsx

```diff
@@ -81,6 +82,9 @@ const UserMenu: React.FC<UserMenuProps> = (props) => {
                 className="rounded"
                 onClick={(e) => {
                   e.preventDefault();
+                  postSignOut().then((res) => {
+                    console.log(res);
+                  });
                   removeCookie("auth-token");
```

Comment:
You should put removeCookie & set location in side the `then` wrapper so that they will only be executed after postSignOut is successed.

A better way to do this is not calling the API be redirect on to a `logout` route which everything is handled by the backend.

Reply:
I dont really understrand the better way.  That's means backend return a redict url？